library/std/Cargo.toml

```diff
@@ -82,6 +82,7 @@ system-llvm-libunwind = ["unwind/system-llvm-libunwind"]
 
 # Make panics and failed asserts immediately abort without formatting any message
 panic_immediate_abort = ["core/panic_immediate_abort", "alloc/panic_immediate_abort"]
+restricted-std = []
```

Comment:
this doesn't really make sense as a feature, using a `feature = ` cfg is a bit weird.. could you put it into the check-cfg table as well?